Formula/litani.rb

```diff
@@ -0,0 +1,59 @@
+class Litani < Formula
```

Comment:
Is this supposed to be `AwsBuildAccumulator`?

Reply:
I would like to keep it as `Litani` as that is the name widely used for this package.